scm/branching-model/feature-toggling.md

```diff
@@ -1,6 +1,8 @@
 # Feature Toggling
 
-Following considerations need to be taken care when a feature toggling approach is considered
+Feature Toggles allow new components and changes to be deloyed in an inactive state and enabled at a time when stakeholders or technical dependencies are ready.
+
+This pattern can decouple the risk and logistics of the deployment itself from the business considerations involved in adopting a new feature, and helps to support the practice of small and frequent deployments.
```

Comment:
Probably worth mentioning that the first feature toggling mechanism that can be implemented is "do we deploy the feature or not". Also, feature toggles are usually transient, we deploy a new feature, we test it then the feature stays. Beyond enabling feature toggling, removing the toggles after a while is something that need to be worked on (note to myself here)

Reply:
Yup, I've included a bullet point at the end about not deploying. Also there was a bullet point there already about removing toggles which is a good reminder